cartographer_ros/cartographer_ros/node_main.cc

```diff
@@ -80,17 +80,31 @@ void Run() {
 }  // namespace cartographer_ros
 
 int main(int argc, char** argv) {
-  // It is important to initialize rclcpp first or copy argv,
-  // because the google commands eat the arguments
-  ::rclcpp::init(argc, argv);
-  google::InitGoogleLogging(argv[0]);
-  google::ParseCommandLineFlags(&argc, &argv, true);
+  // Strip ROS specific arguments
+  std::vector<std::string> non_ros_args = rclcpp::remove_ros_arguments(argc, argv);
```

Comment:
Nit: `#include <string>` and `#include <vector>` at the top of the file, please.

Reply:
No longer using string and vector with new approach